_doc-build-windows/action.yml

```diff
@@ -445,7 +474,7 @@ runs:
       id: set-expected-build-directory
       shell: powershell
       run: |
-        echo "EXPECTED_BUILD_DIR=$(echo 'doc\_build')" | Out-File -FilePath $env:GITHUB_OUTPUT -Encoding utf8 -Append
+        echo "EXPECTED_BUILD_DIR=$(echo '_build')" | Out-File -FilePath $env:GITHUB_OUTPUT -Encoding utf8 -Append
```

Comment:
I am surprised this works. I thought that even if using `cd` in previous every step defaults to the base directory.

Reply:
Yeah... me too.

Reply:
Verified here by @SMoraisAnsys -- the previous location is not kept. We need to check what's going on here.

See https://github.com/ansys/actions/pull/870#issuecomment-2929321191